javascript/src/reducers/layers.js

```diff
@@ -63,6 +63,15 @@ export default (state = [], action) => {
             case TOGGLE_LAYER:
                 index = state.findIndex(layer => layer.name === action.name)
                 return state.slice(0, index).concat([morph(state[index], {displayed: !state[index].displayed}), ...state.slice(index+1)])
+            
+            case START_JOB:
+                let lastRunIndex = state.findIndex(layer => layer.name === 'Last Run')
```

Comment:
Could rename lastRunIndex to "index". (I use index so much in this reducer I already have the variable "index" initialized at the top of the function.)

Reply:
Good call.